packages/react-components/react-combobox/Spec.md

```diff
@@ -143,9 +143,9 @@ Combobox in v9 allows both controlled and uncontrolled selection, as do the corr
 | -------------------- | ------------------- | ------------------- | ------------------ | -------------------- | ------------ |
 | Initial selection    | initialSelectedKeys | defaultSelectedKeys | defaultSelectedKey | defaultSelectedItems | defaultValue |
 | Controlled selection | selectedKeys        | selectedKey         | selectedKey        | selectedItems        | value        |
-| Callback             | onSelect            | onChange            | onChange           | onChange             | onChange     |
+| Callback             | onOptionSelect      | onChange            | onChange           | onChange             | onChange     |
```

Comment:
Would onSelectOption be a better name?  (It reads more smoothly to me)

Reply:
I kinda agree it sounds better in isolation, but I think I'd rather leave it as `onOptionSelect`, since our pattern seems to be `noun-verb`, and leaving the event verb at the end of our event handler names. Some examples include `onCheckedValueChange`, `onOpenChange`, and `onTabSelect`.